### ISSUE TITLE
editor.selectionBackground has very low contrast ratio

### ISSUE THREAD

**Issue by motss** [beastdestroyer/vscode-firefox-quantum-themes#40]
**Describe the bug**
`editor.selectionBackground` needs better contrast ratio. I'm currently using `"editor.selectionBackground": "#2179775c"` which better highlights the selection in the editor.

Of course, feel free to use any better color for the selection.

**Screenshots**
N/A

**Environment:** 
 - Other Data:  N/A
 - Theme Version: 1.1.8

**Additional context**

**Comment by beastdestroyer** [comment]
Hey there, thanks for the concern!

Apologies for the late reply, school has been extremely demanding.

I will get onto this asap.

**Comment by beastdestroyer** [comment]
Should have been done now, thanks!
